engine/src/cmd/collide2/OPC_PlanesTriOverlap.h

```diff
@@ -1,9 +1,30 @@
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 /**
- * Updated by Stephen G. Tuggy 2021-07-03
- * Updated by Stephen G. Tuggy 2022-01-06
- * Updated by Benjamen R. Meyer 2023-05-27
+ * OPC_PlanesTriOverlap.h
```

Comment:
All the files related to http://www.codercorner.com/Opcode.htm are public domain.
For 0.9.x let's hold off on adding GPL to them for the moment. Reviewing the Zip file from there for the 1.3 release it seems there's a couple text files (Readme, TemporalCoherence) that are missing from VS. I'll file a story about that library into its own library as part of #56 but we'll do that on master.

Reply:
Put these under "Public Domain."